budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/pages/budgets/overview/table/BudgetOverviewTable.java

```diff
@@ -86,10 +87,21 @@ private void addTableSummaryLabels(WebMarkupContainer table, IModel<List<BudgetD
     @Override
     public void onEvent(IEvent<?> event) {
         super.onEvent(event);
-        if (event.getPayload() instanceof BudgetTagFilter) {
+        Object payload = event.getPayload();
+        if (payload instanceof BudgetTagFilter) {
             BudgetTagFilter filter = (BudgetTagFilter) event.getPayload();
             FilteredBudgetModel model = (FilteredBudgetModel) getDefaultModel();
             model.setFilter(model(from(filter)));
+        }else if(payload instanceof String){
+            Long remainingFilter;
```

Comment:
This surprised me: If the payload of an event is a string, it has to be a remaining filter value? In the if-clause above, the event was an object. I would have expected that for remaining filter, the payload would be an object as well.